lib/extracker/stories.ex

```diff
@@ -3,6 +3,22 @@ defmodule ExTracker.Stories do
   alias Extracker.Client
 
   @doc """
+  Get a single `story`.
+
+  ## Example
+
+      ExTracker.Stories.find("12345", "67890", client)
+
+  More info at: https://www.pivotaltracker.com/help/api/rest/v5#Story
```

Comment:
@bigardone what do you think about using the endpoint that just requires the `story_id` and not the `project_id`?
https://www.pivotaltracker.com/help/api/rest/v5#stories_story_id_get

The `project_id` isn't required to get a story, and it allow a bit more flexibility for the user.